### PR TITLE
Added new endpoints to index.md for HP PageWide Pro

### PR DESCRIPTION
**Pull request by oleg-andreyev**

### CODE REVIEW

**Comment by manuc66** [comment]
Hi @oleg-andreyev 

Good idea to document another printer!

 I'm wondering if it wouldn't have been a better idea to add those new endooint/call flow in a new .md file that show all the call in chronological order (as it is right now in the document for the HP Officejet 6500A Plus)

**Comment by oleg-andreyev** [comment]
@manuc66 sure, divided into two files.

File: protocol_doc/HP Officejet 6500 E710n-z.md [hunk 1]
```diff
@@ -28,6 +28,10 @@
         - [`GET /Scan/Status`](#get-scanstatus-2)
         - [`GET /EventMgmt/EventTable?timeout=1200`](#get-eventmgmteventtabletimeout1200-1)
         - [`GET /EventMgmt/EventTable?timeout=1192`](#get-eventmgmteventtabletimeout1192)
+        - [`GET /eSCL/ScannerCapabilities`](#get-escl-scanner-capabilities)
```
Comment (manuc66) [inline protocol_doc/HP Officejet 6500 E710n-z.md:31]: This section is no longer in this file

File: protocol_doc/HP PageWide Pro 477dw MFP.md [hunk 2]
```diff
@@ -0,0 +1,808 @@
...
+    - [Untested](#untested)
+      - [`DELETE /eSCL/ScanJobs/893e6fcd-487f-4056-a8c9-a87709b85daf`](#delete-escl-scan-jobs)
+      - [`PUT /eSCL/ScanBufferInfo`](#put-escl-scan-buffer-info)
+<!-- /TOC -->
+
+# Protocol
+
+- Scan to computer for `HP PageWide Pro 477dw MFP` recorder on a macOS (HP Easy Scan) computer.
+- Scan to computer for `HP PageWide Pro 477dw MFP` recorder on a Windows 11 (HP Smart) computer.
+- Used [this pdf](https://mopria.org/MopriaeSCLSpecDownload.php)
```
Comment (manuc66) [inline protocol_doc/HP PageWide Pro 477dw MFP.md:19]: Nice found !

File: protocol_doc/HP PageWide Pro 477dw MFP.md [hunk 3]
```diff
@@ -0,0 +1,808 @@
...
+			<pwg:JobStateReasons>
+				<pwg:JobStateReason>JobCompletedSuccessfully</pwg:JobStateReason>
+			</pwg:JobStateReasons>
+		</scan:JobInfo>
+	</scan:Jobs>
+</scan:ScannerStatus>
+
+```
+
+### `POST /eSCL/ScanJobs`
```
Comment (manuc66) [inline protocol_doc/HP PageWide Pro 477dw MFP.md:555]: Before this, I can't see what does trigger it

According to this page : https://support.hp.com/us-en/document/c05500578 I would expect something that tell the user as touched either `Send` or `Start Scan`

Do you know how the "Scan to Computer" tool knows that it has to request a scan ?

File: protocol_doc/HP PageWide Pro 477dw MFP.md [hunk 4]
```diff
@@ -0,0 +1,808 @@
+<!-- TOC -->
```
Comment (manuc66) [inline protocol_doc/HP PageWide Pro 477dw MFP.md:1]: Thanks for splitting the file

**Comment by manuc66** [comment]
@oleg-andreyev you might be interested in this new dev : https://github.com/manuc66/node-hp-scan-to/pull/1331

**Pull request merged**